config/email_service_provider.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  provider: <%= ENV["EMAIL_SERVICE_PROVIDER"] || "PSEUDO" %>
```

Comment:
I know this is how other config files are done, but maybe we should be using `ENV.fetch` here.

Reply:
any particular reason? won't that raise an exception if the key is not found?
My intention behind this env var is that it should be used for override only and not be set by default.....

Reply:
Oh sorry, I meant like `ENV.fetch("EMAIL_SERVICE_PROVIDER", "PSEUDO")` and similar. It's just a personal preference really, I refer that way of doing it.

Reply:
I think we should keep it consistent with the rest of the app.....either that or change them all.